rotkehlchen/db/evmtx.py

```diff
@@ -67,6 +68,8 @@
 
 
 class DBEvmTx:
+    # Index in the SQL result tuple where authorization fields (nonce, delegated_address) begin  # noqa: E501
+    authorization_data_start_index = 13
```

Comment:
This gets created again and again for each class instance. Do not do that. Make it into a file lever constant :Final